packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -816,45 +836,41 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
         }
     }
 
-    protected _updateRotationSpheres() {
-        const rotateSpheres = this._rotateSpheresParent.getChildMeshes();
+    protected _updateRotationAnchors() {
+        const rotateAnchors = this._rotateAnchorsParent.getChildMeshes();
         for (let i = 0; i < 3; i++) {
             for (let j = 0; j < 2; j++) {
                 for (let k = 0; k < 2; k++) {
                     const index = i * 4 + j * 2 + k;
                     if (i == 0) {
-                        rotateSpheres[index].position.set(this._boundingDimensions.x / 2, this._boundingDimensions.y * j, this._boundingDimensions.z * k);
-                        rotateSpheres[index].position.addInPlace(new Vector3(-this._boundingDimensions.x / 2, -this._boundingDimensions.y / 2, -this._boundingDimensions.z / 2));
-                        rotateSpheres[index].lookAt(
-                            Vector3.Cross(rotateSpheres[index].position.normalizeToNew(), Vector3.Right()).normalizeToNew().add(rotateSpheres[index].position)
-                        );
+                        rotateAnchors[index].position.set(this._boundingDimensions.x / 2, this._boundingDimensions.y * j, this._boundingDimensions.z * k);
+                        rotateAnchors[index].position.addInPlace(new Vector3(-this._boundingDimensions.x / 2, -this._boundingDimensions.y / 2, -this._boundingDimensions.z / 2));

```

Comment:
Related to my other comment - feels like this can be completely skipped?

```javascript
rotateAnchors[index].position.set(this._boundingDimensions.x / 2 - this._boundingDimensions.x / 2, this._boundingDimensions.y * j - this._boundingDimensions.y / 2, this._boundingDimensions.z * k - this._boundingDimensions.z / 2);
```